src/sonarwhal-theme/layout/partials/scan-result-item.hbs

```diff
@@ -13,17 +13,14 @@
     <div class="rule-result--details__body">
         <p class="{{../status}}-badge uppercase-text">{{../status}}</p>
         <p>
-            {{message}}
+            {{{linkify message}}}
         </p>
         {{#if resource}}
-        <div class="rule-result__code">
-
-            <p>
-                <a target="_blank" rel="noopener noreferrer" href="{{resource}}">
-                    {{cutUrlString resource}}{{normalizePosition location.line}}{{normalizePosition location.column}}
-                </a>
-            </p>
-        </div>
+        <p>
+            <a target="_blank" rel="noopener noreferrer" href="{{resource}}">
```

Comment:
Aren't we [taking care of the `rel="noopener noreferrer"` part automatically?](https://github.com/sonarwhal/sonarwhal.com/commit/051da29a7f6c4785cc182b3471214d0d50b18f57).

Cc @molant

Reply:
@alrra I thought so too but couldn't find the script. It should be a hexo helper. I wonder if it was removed because it's not compatible with the new renderer? @molant

Reply:
The `noopener noreferrer` is taking care automatically via [`remarkable-extlink`](https://github.com/sonarwhal/sonarwhal.com/blob/67b927f01f84c284aac128752eb28df8af4e4b79/src/sonarwhal-theme/scripts/custom-renderer.js#L4). But this only applies to `markdown` files so we need to manually do it for `hbs` templates.
Even with the previous solution we would have to do this.